tests/unit/test-class-wc-payments-account.php

```diff
@@ -3174,6 +3174,73 @@ public function test_get_tracking_info() {
 		$this->assertSame( $expected, $this->wcpay_account->get_tracking_info() );
 	}
 
+	public function test_get_recommended_payment_methods_unsupported_country() {
+		$this->assertSame( [], $this->wcpay_account->get_recommended_payment_methods( 'unsupported_country' ) );
```

Comment:
`unsupported_country` works but its a poor replacement for an actual country code (two letters). If that method validates the country code by length, we will never reach the stage of actually checking for WooPayments supported countries. I think `XZ` is a better bogus country code.

Reply:
the ideea was to be clear what is that id, ATM the method desn't check for the code length. anyway, changed it in [3af3993](https://github.com/Automattic/woocommerce-payments/pull/9825/commits/3af399332f1bc9159887a0f09db97ea47fa75fcb)

Reply:
> the ideea was to be clear what is that id
But it is not an ID; it is quite a special string.

> ATM the method desn't check for the code length
We shouldn't test the internals of the method. We should strive to test the expected outputs when given a set of inputs.